rules.ini

```diff
@@ -201,7 +201,7 @@ NVIDIA_PhysX = (?:^|/)PhysX(?:_64|_32|3_x64|3_x86|Core|Loader)\.dll$
 NWJS = (?:^|/)nw\.(?:dll|pak)$
 OculusXRPlugin = (?:^|/)OVRPlugin\.dll$
 Oodle = (?:^|/)oo2(?:core|net|ext)\_[3-9]\_win(?:32|64)\.dll$
-OpenAL = (?:^|/)OpenAL(?:32|64)\.dll$
+OpenAL = (?:^|/)OpenAL(?:32|64)(?:_bundled)\.dll$
```

Comment:
```suggestion
OpenAL = (?:^|/)OpenAL(?:32|64)(?:_bundled)?\.dll$
```

Reply:
oh well, i'll use the one you suggested instead